.travis.yml

```diff
@@ -8,10 +8,8 @@ python:
 
 jobs:
   include:
-  - name: "Testing ansible playbook `deploy-discourse.yml`"
-    env: TEST_ROLE=discourse
-  - name: "Testing ansible playbook `deploy-matterbridge.yml`"
-    env: TEST_ROLE=matterbridge
+  - name: "Testing ansible playbook `deploy-update.yml`"
+    env: TEST_ROLE=update
```

Comment:
Note, I intentionally test only this role as the other role tests are broken in master.

I have fixed them in: https://github.com/PrivacyLx/devops/pull/37

Reply:
I see. So we merge those first